content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
+weight: 201
+menu:
+  v2_0:
+    name: Enabling HTTPS
+    parent: Security & authorization
+v2.0/tags: [security, authentication, tls, https]
+---
+
+Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
```

Comment:
Enable Transport Layer Security (TLS) and use the HTTPS protocol to secure communication between clients and InfluxDB.